From e0fd0904b8542cc07fb58777f9ff7c63dd7b3bda Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:09:36 -0400
Subject: [usb] pci_quirks: fix sparse 'symbol not declared' warning

Message-id: <1410887463-20674-505-git-send-email-dzickus@redhat.com>
Patchwork-id: 95597
O-Subject: [RHEL7 PATCH 504/591] usb: pci_quirks: fix sparse 'symbol not declared' warning
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 342a74934197386e065e8ef00014e6f0cb5effe6
Author: Konrad Zapalowicz <bergo.torino@gmail.com>
Date:   Tue May 27 23:09:14 2014 +0200

    usb: pci_quirks: fix sparse 'symbol not declared' warning

    This commit fixes the following sparse warning:

    drivers/usb/host/pci-quirks.c:
        - 252: warning: symbol 'usb_hcd_amd_remote_wakeup_quirk' was not
          declared. Should it be static?

    This function is exported so the fix was to add it's declaration to the
    header file.

    Signed-off-by: Konrad Zapalowicz <bergo.torino@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/host/pci-quirks.h b/drivers/usb/host/pci-quirks.h
index 638e88f..c622ddf 100644
--- a/drivers/usb/host/pci-quirks.h
+++ b/drivers/usb/host/pci-quirks.h
@@ -5,6 +5,7 @@
 void uhci_reset_hc(struct pci_dev *pdev, unsigned long base);
 int uhci_check_and_reset_hc(struct pci_dev *pdev, unsigned long base);
 int usb_amd_find_chipset_info(void);
+int usb_hcd_amd_remote_wakeup_quirk(struct pci_dev *pdev);
 bool usb_amd_hang_symptom_quirk(void);
 bool usb_amd_prefetch_quirk(void);
 void usb_amd_dev_put(void);
-- 
1.7.1